doc/tutorials/introduction/crosscompilation/crosscompile_with_multiarch.markdown

```diff
@@ -0,0 +1,573 @@
+Cross compilation with Ubuntu/Debian{#tutorial_crosscompile_with_multiarch}
+====================================
+
+@prev_tutorial{tutorial_arm_crosscompile_with_cmake}
+@next_tutorial{tutorial_building_tegra_cuda}
+
+[TOC]
+
+|    |    |
+| -: | :- |
+| Original author | Kumataro |
+| Compatibility   | Ubuntu >=23.04 |
+|^                | OpenCV >=4.8.0 |
+
+@warning
+This tutorial can contain obsolete information.
+
+What is "MultiArch"
+-------------------
+OpenCV may use a lot of 3rdparty libraries for video and image decoding, rendering, acceleration and complex math algorithms. The 3rdparty components are found by CMake on the build host Cross-compilation allows to build OpenCV for foreign architecture or OS, but we loose that large world of components and have to cross-compile each dependency separately and point to it during OpenCV build.
+
+Debian/Ubuntu MultiArch helps to fix this. It allows to install several foreign architecture libraries on host system and use them during OpenCV dependencies resolution.
+
+@warning
+- Following these steps will make your Linux environment a little dirty.
+  If possible, it is better to use VMs or Container(e.g. Docker).
+- This tutorial expects host and target uses same ubuntu version.
+   Do not use/mix different versions for external library dependency.
+  - Good: Host and Target are 23.04.
+  - Good: Host and Target are 23.10.
+  - Not Good: Host is 23.04, and Target is 23.10.
+  - Not Good: Host is 23.10, and Target is 23.04.
+- This tutorial may be able to use for debian(and Raspberry Pi OS).
```

Comment:
Debian with capital letter here and in the following places. The same for Ubuntu.

Reply:
Thank you, I replaced from "ubuntu/debian" to "Ubuntu/Debian" at all.